src/Compilers/VisualBasic/Test/Syntax/Parser/ParseErrorTests.vb

```diff
@@ -857,8 +857,8 @@ End Module
                 End Sub
             End Module
                 </file>
-        </compilation>).VerifyDiagnostics(Diagnostic(ERRID.ERR_RequiredConstExpr, "New Integer(1, 1) {{1, 2}, {2, 3}}"),
-            Diagnostic(ERRID.ERR_OverloadWithDefault2, "foo").WithArguments("Public Sub foo([i As Integer(*,*)])", "Public Sub foo([i As Integer(*,*) = Nothing])"))
```

Comment:
Note: Does this need to be two separate test, the original (without feature) and one with the feature enable.